access-checkout/src/test/java/com/worldpay/access/checkout/ui/AccessEditTextTest.kt

```diff
@@ -0,0 +1,72 @@
+package com.worldpay.access.checkout.ui
+
+import android.graphics.Color
+import android.text.InputType
+import android.text.method.DigitsKeyListener
+import com.worldpay.access.checkout.R
+import com.worldpay.access.checkout.testutils.CardNumberUtil.visaPan
+import kotlin.test.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.shadows.ShadowInstrumentation
+
+@RunWith(RobolectricTestRunner::class)
+class AccessEditTextTest {
+
+    private val context = ShadowInstrumentation.getInstrumentation().context
+
+    private lateinit var accessEditText: AccessEditText
+
+    @Before
+    fun setUp() {
+        accessEditText = AccessEditText(context)
+    }
+
+    @Test
+    fun `should return current text colour when called`() {
```

Comment:
This tests covers both setting the color property and accessing the color property - can you rename it so it reflects that? (and same for others below where applicable)